remoteworkitem/remoteworkitem_test.go

```diff
@@ -80,7 +81,7 @@ func TestGitHubIssueMapping(t *testing.T) {
 	}
 
 	workItemMap := WorkItemKeyMaps[ProviderGithub]
-	remoteWorkItem := RemoteWorkItem{ID: "xyz", Content: []byte(content)}
+	remoteWorkItem := TrackerItem{Item: string(content[:]), RemoteItemID: "xyz", BatchID: "bxyz", TrackerQueryID: uint64(0)}
```

Comment:
Sure @aslakknutsen , I agree - I'll keep that in mind when I name variables in future.
I've made the necessary name changes - good to go!